pkg/servicelb/controller.go

```diff
@@ -398,6 +406,20 @@ func (h *handler) newDaemonSet(svc *core.Service) (*apps.DaemonSet, error) {
 		},
 	}
 
+	if ipv6Switch {
+		// Add security context to enable ipv6 forwarding
+		securityContext := &core.PodSecurityContext{
+			Sysctls: []core.Sysctl{
+				{
+					Name:  "net.ipv6.conf.all.forwarding",
+					Value: "1",
+				},
+			},
+		}
+
+		ds.Spec.Template.Spec.SecurityContext = securityContext
+	}
+
```

Comment:
![image](https://user-images.githubusercontent.com/1033444/141158303-68a2a7f4-4bd9-49d7-841f-47832147c205.png)

---

(but for whitespace :rofl: )